stack.yml

```diff
@@ -67,21 +75,25 @@ functions:
     lang: go
     handler: ./garbage-collect
     image: alexellis2/garbage-collect:0.3.1
+    labels:
+      openfaas-cloud: "1"
     environment:
       write_debug: true
       read_debug: true
-      read_timeout: 30
-      write_timeout: 30
+      read_timeout: 30s
+      write_timeout: 30s
     environment_file:
       - gateway_config.yml
     secrets:
       - basic-auth-user
       - basic-auth-password
 
-  git-event:
+  github-event:
     lang: go
-    handler: ./git-event
-    image: alexellis2/git-event:0.3.1
+    handler: ./github-event
+    image: alexellis2/github-event:0.4.1
+    labels:
+      openfaas-cloud: "1"
```

Comment:
Just out of curiosity, for what this label will be used?

Reply:
Filtering and network policy